CoseSignTool.Tests/MainTests.cs

```diff
@@ -214,4 +214,175 @@ public void ReturnsHelpRequestedWhenNoOptionsAfterVerb()
         string[] args = [ "sign" ];
         CoseSignTool.Main(args).Should().Be((int)ExitCode.HelpRequested);
     }
-}
\ No newline at end of file
+
+    [TestMethod]
+    public void SignWithIntegerHeadersSuccess()
+    {
+        string integerHeadersFile = FileSystemUtils.GenerateHeadersFile(@"[{""label"":""created-at"",""value"":1723588348,""protected"":true}]");
+        string payloadFile = FileSystemUtils.GeneratePayloadFile();
+
+        // sign with integer headers
+        string[] args1 = ["sign", @"/p", payloadFile, @"/pfx", PrivateKeyCertFileChainedWithPassword, @"/pw", CertPassword, @"/ep", @"/ih", integerHeadersFile];

```

Comment:
Any particular reason you're using the cert with the password for these?
Not a problem, just curious.

Reply:
None. Most probably I followed the behavior in one of the tests in the file and ended up with it.